dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Microsoft.ApplicationModel.Resources.vcxproj

```diff
@@ -5,6 +5,7 @@
   <Import Project="..\..\packages\Microsoft.SourceLink.Common.1.1.0-beta-21055-01\build\Microsoft.SourceLink.Common.props" Condition="Exists('..\..\packages\Microsoft.SourceLink.Common.1.1.0-beta-21055-01\build\Microsoft.SourceLink.Common.props')" />
   <Import Project="..\..\packages\Microsoft.Windows.CppWinRT.2.0.210806.1\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\..\packages\Microsoft.Windows.CppWinRT.2.0.210806.1\build\native\Microsoft.Windows.CppWinRT.props')" />
   <Import Project="..\..\..\..\WindowsAppSDK.Build.Cpp.props" />
```

Comment:
>WindowsAppSDK.Build.Cpp.props [](http://example.com/codeflow?start=31&length=29)

wondering why this is under \dev.

Reply:
It's no longer under `\dev` anymore. 🙂

Reply:
i am talking about the one above yours "WindowsAppSDK.Build.Cpp.props"" :)

Reply:
Ah, yeah good question.. I'm not sure why that one is under \dev either...